ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -546,29 +548,29 @@ define([
       }
     },
 
-    isUtmLatLonValid: function(lat) {
-      return lat <= 84 && lat >= -80
+    isUtmUpsLatLonValid: function({lat} = {lat : NaN}) {
```

Comment:
UICOP: we are giving too much **latitude** to this function, perhaps we simplify this back to just passing lat?